Content/StructureResolver.php

```diff
@@ -95,6 +103,10 @@ public function resolve(
             $data['view'][$property->getName()] = $contentView->getView();
         }
 
+        foreach ($this->structureResolverExtensions as $structureResolverExtension) {
+            $data[$structureResolverExtension->getKey()] = $structureResolverExtension->resolve($requestedStructure, $locale);
```

Comment:
As written in the other comments we need to go here with a `array_merge/array_replace` as there will be multiple extensions adding a breadcrumb for example.  
```suggestion
            $data = \array_replace($data, $structureResolverExtension->resolve($requestedStructure, $locale));
```